model/patch/patch.go

```diff
@@ -194,6 +194,8 @@ type Patch struct {
 	// LocalModuleIncludes is only used for CLI patches to store local module changes.
 	// Not stored in the database since the DB patch should already include changes from this module.
 	LocalModuleIncludes []LocalModuleInclude `bson:"-"`
+	// RepeatPatchID is used to store the ID of the patch that this patch is a repeating from the CLI.
+	RepeatPatchID string `bson:"-"`
```

Comment:
Could we name this something else a little more abstract to reuse for patches that should use the module revisions of another patch?